crd/crd.go

```diff
@@ -81,21 +91,15 @@ func NewDatabaseCRD() *apiextv1.CustomResourceDefinition {
 											Minimum:     floatptr(20),
 											Maximum:     floatptr(64000),
 										},
-										"MaxAllocatedSize": {
```

Comment:
This field has been added by https://github.com/sorenmat/k8s-rds/commit/75301c210b870deb52cd904193494f2b0fec28e5 on Nov 5, 2020.

It's been kind of mandatory field since it's been added. Not sure why our `db.yaml`s have been working. Maybe very old image has been running until recently?

Excluding recent changes by Jun and Kenji, our activity on our Helm chart has stopped before the time the field was added: https://github.com/cloudnatix/k8s-rds-helm-chart/commits/master

Rather than changing the field to support not enabling the autoscaling (i.e. not setting `MaxAllocatedStorage` on  `CreateDBInstanceInput` when the custom resource doesn't have the field set), just removed the field completely for now to save time...